### ISSUE TITLE
@types/relay-runtime const enum FragmentReference

### ISSUE THREAD

**Issue by sgwilym** [DefinitelyTyped/DefinitelyTyped#33067]
- [x] I tried using the `@types/xxxx` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @voxmatt @alloy 

When trying to use the latest version of @types/relay-runtime I get the following error:

```
Failed to compile.

/Users/gwil/Projects/grokument-browser/node_modules/@types/relay-runtime/index.d.ts
TypeScript error: Ambient const enums are not allowed when the '--isolatedModules' flag is provided.  TS1209

    82 | // Using `enum` here to create a distinct type and `const` to ensure it doesn’t leave any generated code.
    83 | // tslint:disable-next-line:no-const-enum
  > 84 | export const enum FragmentReference {}
       |                   ^
    85 | 
    86 | export interface OperationBase {
    87 |     variables: object;
```

Excluding any usage of the `isolatedModules` flag excludes the usage of this package with <s>projects that use Typescript via Babel</s> create-react-app (https://github.com/facebook/create-react-app/issues/5738). Would it be possible to find another way to define this type?

**Comment by alloy** [comment]
We actually ended up not even using that type for the artifacts emitted by https://github.com/relay-tools/relay-compiler-language-typescript, so I think we can just remove it. Care to PR that?

Btw, slightly offtopic, it's not true that all TS projects that use Babel for compilation need this flag. We use Babel to compile our TS with no problem.

**Comment by sgwilym** [comment]
Huh, sorry for spreading misinformation in that case! Glad that it can simply be removed, I'll put together the PR now.

**Comment by alloy** [comment]
Fixed in #33072
